dali/operators/generic/cast.cc

```diff
@@ -73,4 +73,11 @@ DALI_SCHEMA(Cast)
     .SupportVolumetric()
     .AddTypeArg("dtype", R"code(Output data type.)code");
 
+DALI_SCHEMA(CastLike)
+    .DocStr("Cast tensor to the type of another tensor.")
```

Comment:
How about:
```suggestion
    .DocStr("Cast the first tensor to the type of the second tensor.")
```
to make it more explicit which of the inputs gets cast?